src/main/webapp/dev/js/common/feedbackResponseComments.js

```diff
@@ -165,7 +166,12 @@ function enableHoverToDisplayEditOptions() {
 }
 
 function showResponseCommentAddForm(recipientIndex, giverIndex, qnIndex, sectionIndex) {
-    const id = `${sectionIndex === undefined ? '' : `-${sectionIndex}`}-${recipientIndex}-${giverIndex}-${qnIndex}`;
+    let id;
+    if (giverIndex === null && sectionIndex === undefined) {
```

Comment:
Can we write code just for the feedback participant side?

Reply:
Okay, will do that